src/routes.ts

```diff
@@ -1,6 +1,11 @@
 import { StackNavigationProp } from '@react-navigation/stack';
 import { RouteProp } from '@react-navigation/native';
 
+type FareResultsParam = {
+  departStation: string | null;
```

Comment:
same as above, I don't think it really makes any sense for these to be nullable. You should force the code to know what its departure/arrival station is to be able to show fare results